SSToastMessage/Example/SSToastMessage/ContentView.swift

```diff
@@ -105,6 +112,17 @@ struct ContentView: View {
                 }) {
                     self.createBottomFloaterView()
                 }
+                .present(isPresented: $showLeftToastView, type: .leftRightToastView, position: .bottom, duration: 3, leftToastMessage: showLeftToastView, onTap: {
```

Comment:
If the alert is displayed using `isPresented` and `type` is also defined, why do we need `leftToastMessage` or `rightToastMessage` parameter?